tests/test_download.py

```diff
@@ -164,32 +192,26 @@ def test_download_url_to_tempfileobj_and_urls(self):
     download_file = download.safe_download
     unsafe_download_file = download.unsafe_download
 
-    self.assertRaises(securesystemslib.exceptions.FormatError,
-                      download_file, None, self.target_data_length)
-
-    self.assertRaises(tuf.exceptions.URLParsingError,
-                      download_file,
-                      self.random_string(), self.target_data_length)
+    with self.assertRaises(securesystemslib.exceptions.FormatError):
+      download_file(None, self.target_data_length, self.fetcher)
 
-    url = 'http://localhost:' \
+    url = 'http://127.0.0.1:' \
```

Comment:
Nit: a change like this in future might be easier if we define the local URL as a test level constant

Reply:
A suggestion in 5d3133d.